### ISSUE TITLE
jar version dependency conflict warning are globally omitted when Jars.lock exists

### ISSUE THREAD

**Issue by colinsurprenant** [mkristian/jar-dependencies#46]
independently of the content of the `Jars.lock` file (it can be empty) all warnings are globally disabled [I believe here](https://github.com/mkristian/jar-dependencies/blob/fd41b794529e3e874f974f28d8886e8d9e94c87d/lib/jar_dependencies.rb#L241).

I think that the proper behaviour would be to not produce warnings only specifically for the jars explicitly included in the `Jars.lock` ?

Disabling all warnings upon `Jars.lock` prevents getting the notifications for other potential versions conflicts+choice when we only force a specific jar version for example.

**Comment by colinsurprenant** [comment]
@mkristian WDYT?

**Comment by mkristian** [comment]
yes totally turning off warnings is going to far. I definitely want to ignore warnings on those jar loaded via Jars.lock and right the version space outside the Jars.lock shall be treated as if there is no lock file.
